### PR TITLE
Adds more items to OPFOR

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Adds the following:

- Hacked AI module
- Powersink
- Heister bundle (It's payday, fellas)
- Gorilla Cubes
- Meat hook
- Chameleon Module
- Toolbox Sentry Gun
- Voice-Muffling Balaclava

## How This Contributes To The Skyrat Roleplay Experience
Less stuff you have to manually ask admins for

## Changelog

:cl:
qol: More opfor devices available
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/opposing_force/code/equipment/utility.dm [hunk 1]
```diff
@@ -150,10 +150,38 @@
...
+			active gravitational singularities or tesla balls towards it. This will not work when the engine is still \
+			in containment. Because of its size, it cannot be carried. Ordering this \
+			sends you a small beacon that will teleport the larger beacon to your location upon activation."
+	admin_note = "Generally requires a Supermatter Delamination to be effective."
+
 /datum/opposing_force_equipment/gear/cloak_mod
 	item_type = /obj/item/mod/module/stealth/ninja
 	description = "An upgraded MODsuit cloaking module stolen from the Spider Clan's finest. Consumes less power than the standard, but is obviously illegal."
 
+/datum/opposing_force_equipment/gear/flamethrower_mod
```
Comment (YakumoChen) [inline modular_skyrat/modules/opposing_force/code/equipment/utility.dm]: uh zone, this aint it

File: modular_skyrat/modules/opposing_force/code/equipment/utility.dm [hunk 2]
```diff
@@ -150,10 +150,38 @@
...
+	item_type = /obj/item/mod/module/chameleon
+	description = "A module that enables the user to disguise their MODsuit as any other type. Only works while undeployed."
```
Comment (ghost) [inline modular_skyrat/modules/opposing_force/code/equipment/utility.dm]: these dont match, I assume its just bc this is a draft PR and you're going to change this later, but important to have the fat reminder lest we cause jank

**Comment by ghost** [comment]
as for the powersink/power beacon... why?
How can a Powersink, one of the grossest items, contribute to OpFors and roleplay? All its ever used for is to turn off the station, and hidden in the scummiest places possible. And the power beacon just doesnt work, as we never have teslas nor singulos.

**Comment by tf-4** [comment]
The power beacon does work but ONLY with a singulo or tesla delam, which is beyond niche and would require admin scrutiny to the point that it makes more sense as an "off menu" item anyway.

Agree on power sink also, that shit sucks for RP

**Comment by YakumoChen** [comment]
@Zonespace27 add voice muffling balaclavas to clothing

**Comment by Zonespace27** [comment]
> @Zonespace27 add voice muffling balaclavas to clothing

Sure

**Review by Gandalf2k15 — APPROVED** [review]
Ok

**Pull request merged**